pkg/kubelet/kubelet.go

```diff
@@ -1350,6 +1350,9 @@ func (kl *Kubelet) GeneratePodHostNameAndDomain(pod *api.Pod) (string, string, e
 // GenerateRunContainerOptions generates the RunContainerOptions, which can be used by
 // the container runtime to set parameters for launching a container.
 func (kl *Kubelet) GenerateRunContainerOptions(pod *api.Pod, container *api.Container, podIP string) (*kubecontainer.RunContainerOptions, error) {
+	if pod.Status.PodIP == "" {
```

Comment:
We should probably pass through PodIP to ExpandConfigMap (there must have been some reason the code was structured this way). Hopefully someone will point out why :)